### DISCUSSION TITLE
Compilation issues

### DISCUSSION THREAD

**Discussion by Emmaker** [ImageMagick/ImageMagick#7496] [Development]
### ImageMagick version

7.1.1-35

### Operating system

Windows

### Operating system, version and so on

10.0.19045 Build 19045

### Description

When attempting to compile the application, the following error appears when trying to compile the final product.
![image](https://github.com/user-attachments/assets/6915df6a-fd17-4048-9495-a89b8f376dcf)
This is after cloning the repositories and configuring the build.

### Steps to Reproduce

1. Clone the ImageMagick-Windows repository.
2. Run CloneRepositories.IM7.cmd
3. Compile Configure.exe
4. Use default settings, as shown here: 
![image](https://github.com/user-attachments/assets/88703323-8343-4d59-a901-2f33c8c18bab)
5. Attempt to compile ImageMagick

**Comment by dlemstra** [comment]
Moving this to a discussion because this is probably something in your local setup. It looks like you are mixing 32 bit and 64 bit?

**Reply by Emmaker** [reply]
How would I go about not mixing 32 bit and 64 bit? I put my exact actions up to this point as the steps to reproduce, so if you could correct what I did wrong that would be helpful.

**Reply by dlemstra** [reply]
It seems that I misunderstood you. By the application you are talking about the ImageMagick application and not your own application. After reading your error message again it seems that you are trying to run the `aom` project. That project is set as the default start project because it is the first project but you cannot start it.

**Reply by Emmaker** [reply]
Ah, I believe I understand now what my mistake was. I assume I need to build all projects in the solution, rather than just the top project which I had previously assumed was all I needed to build?

**Reply by Emmaker** [reply]
After doing exactly that, it has compiled successfully and magick.exe (along with many other executables) has been placed inside the Output/bin folder. Is there an estimate on when the site will be back up so I can install ImageMagick through scoop instead?

**Reply by dlemstra** [reply]
The site will be back when it's back. It is down due to power issues and we don't know the exact time when this will be restored.

**Reply by Emmaker** [reply]
Alright. Thank you for your help.
